.gradient/prepare-datasets.sh

```diff
@@ -36,8 +36,13 @@ echo "Starting preparation of datasets"
 # symlink exe_cache files
 exe_cache_source_dir="${PUBLIC_DATASETS_DIR}/poplar-executables-pytorch-3-1-1"
 symlink-public-resources "${exe_cache_source_dir}" $POPLAR_EXECUTABLE_CACHE_DIR
+
+gptj_cache_source_dir="${PUBLIC_DATASETS_DIR}/poplar-executables-3-1-gptj"
+symlink-public-resources "${gptj_cache_source_dir}" "$POPLAR_EXECUTABLE_CACHE_DIR/gptj"
 # Symlink squad
 symlink-public-resources "${PUBLIC_DATASETS_DIR}/squad" "${HF_DATASETS_CACHE}/squad"
+symlink-public-resources "${PUBLIC_DATASETS_DIR}/glue" "${HF_DATASETS_CACHE}/glue"
+symlink-public-resources "${PUBLIC_DATASETS_DIR}/gptj-6b-checkpoints" "${CHECKPOINT_DIR}"
```

Comment:
Checkpoints managed by transformers go to the automatically set transformers directory:

```suggestion
symlink-public-resources "${PUBLIC_DATASETS_DIR}/gptj-6b-checkpoints" "${TRANSFORMERS_CACHE}"
```